core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/stacks/response/tags/TagsV4Response.java

```diff
@@ -42,4 +44,12 @@ public void setUserDefined(Map<String, String> userDefined) {
     public void setDefaults(Map<String, String> defaults) {
         this.defaults = defaults;
     }
+
+    @Override
+    public String getTagValue(String key) {
+        return Optional.ofNullable(application.get(key))
+                .or(() -> Optional.ofNullable(userDefined.get(key)))
```

Comment:
question: which has a higher preference, the user-defined tags, or application tags?